README.md

```diff
@@ -58,7 +60,119 @@ Master and Worker nodes:
 2. Create a stack in Heat by passing the template, environment file, and parameters:
 ```
   $ openstack stack create -t caasp-stack.yaml -e caasp-environment.yaml \
-    --parameter image=CaaSP-2 caasp-stack
+    --parameter image=CaaSP-3 caasp-stack
+```
+
+### INSTALL TEMPLATES FROM COMMAND LINE (FOR MULTIPLE MASTER NODES)
+
+1. Specify the appropriate flavor and network settings in caasp-multi-master-environment.yaml file.
+2. Set master_count to more desired number in caasp-multi-master-environment.yaml file
+e.g.
+
+```
+master_count: 3
+```
+Note: The master count has to be set to odd number of nodes  e.g. 1, 3, 5 and so on
+
+3. In a multi master deployment, load balancer is created which points to multiple master nodes for ports
+   32000 and 6443. By default the load balancer provider is 'haproxy'.
+   You can switch the load balancer provider to 'octavia' by changing the master_lb_provider in caasp-multi-master-environment.yaml file
+   e.g.
+
+```
+master_lb_provider: haproxy
+    ```
+    Note: To verify if load balance with octavia provider or haproxy provider is working correctly in Openstack installation
+    you can try and create load balancer manually and check if the provisioning_status changes to 'Active'
+    e.g.
+```
+neutron lbaas-loadbalancer-create --name test-lb --provider octavia  <SUBNET_ID>
+OR
+neutron lbaas-loadbalancer-create --name test-lb --provider haproxy  <SUBNET_ID>
+
+neutron lbaas-loadbalancer-show <LOAD_BALANCER_ID>
+OR
+Openstack CLI:
+openstack loadbalancer show <LOAD_BALANCER_ID>
+```
+Note: Octavia is default load balancer provider in SUSE Openstack Cloud, but it needs a some prerequisites to be setup like, for example
+amphora image should be available in Glance. Please refer to documentation in "Installing with
+Cloud Lifecyle Manager" guide in Post Installation->Configuring Loadbalancer as a Service->Section 31.3
+[Setup of prerequisites](https://www.suse.com/documentation/suse-openstack-cloud-8/book_installation/data/sect1_51_chapter_book_installation.html)
+for instructions on how to setup prerequisites which includes steps on "Installing the Amphora Image", "Register the image", and also
+steps on "Setup network, subnet, router, security and IP's" to set up a test "lb_net1" network with "lb_subnet1" subnet.
+e.g.
+```
+openstack network create lb_net1
```

Comment:
Also  can be added in code style.

Reply:
thx! I think there were some missing back ticks